gradle.properties

```diff
@@ -8,7 +8,7 @@ readable_name=FTB Chunks
 maven_group=dev.ftb.mods
 mod_author=FTB Team
 
-mod_version=2101.1.12
+mod_version=2101.1.13
```

Comment:
This shouldn't be necessary; 2101.1.11 is the current release version so .12 is already dev.

Also, I find it best to keep commits affecting the build (e.g. mod version) as a separate commit. Mixing them causes problems with actions like cherry-picking.